Gemfile

```diff
@@ -9,7 +9,7 @@ return unless Plugin.installed?("bundler-multilock")
 
 Plugin.send(:load_plugin, "bundler-multilock")
 
-lockfile active: RUBY_VERSION >= "2.7" do
+lockfile active: RUBY_VERSION >= "2.7" && !Object.const_defined?(:JRUBY_VERSION) do
```

Comment:
you can use `RUBY_ENGINE` for this condition

Reply:
I'm not really sure why it stopped working when I changed to RUBY_ENGINE == "jruby" check. I'm not familiar with dealing with bundler :(